src/main/java/org/eclipse/uprotocol/cloudevent/factory/CloudEventFactory.java

```diff
@@ -83,10 +73,10 @@ static CloudEvent request(String applicationUriForRPC,
     static CloudEvent response(String applicationUriForRPC,
                                String serviceMethodUri,
                                String requestId,
-                               Any protoPayload,
+                               Optional<Any> protoPayload,
```

Comment:
It is very possible that a method invocation does not pass anything. It could be a method like `UCode start();` and `UCode stop();` we cannot force service developers that they have to return anything.